frontend/package.json

```
@@ -5,6 +5,8 @@
   "type": "module",
   "scripts": {
     "start": "vite",
+    "dev": "vite",
+    "dev:local": "vite --host localhost",
```

```suggestion
    "start:local": "vite --host localhost",
```

No need for standalone `dev`.
`dev:local` should be `start:local` as for convenience.